client/extensions/woocommerce/app/dashboard/manage-external-view.js

```diff
@@ -28,21 +29,24 @@ class ManageExternalView extends Component {
 	render = () => {
 		const { site, translate } = this.props;
 
-		const title = translate( 'Manage your store' );
+		const title = translate( 'Managing your store' );
 		const line = translate(
-			'Stores in your country are managed directly on your site. ' +
-				'We will let you know when you can manage your store here.'
+			'Howdy! It looks like your store is located in a country that we can not fully ' +
+				'support in this interface. Store setup and management will take place in wp-admin, ' +
+				'our classic admin interface. We are actively working on adding broader support and ' +
+				"we'll let you know when you can manage your store here!"
 		);
 
 		const actionURL = site.URL + '/wp-admin/edit.php?post_type=shop_order';
 		const action = (
-				<a
+				<ExternalLink
+					icon={ true }
```

Comment:
Minor but this can read just `icon` no need for the `true` I believe.

Reply:
True dat (see what I did there?) https://github.com/Automattic/wp-calypso/pull/21567/commits/8a39ea3dd8fb9525ed86749edb1a31c9814a8683